bindings_ffi/examples/MainActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.xmtpv3_example
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.TextView
+import com.example.xmtpv3_example.R.id.selftest_output
+import kotlinx.coroutines.runBlocking
+
+// An example Android app testing the end-to-end flow through Rust
+// Run setup_android_example.sh to set it up
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        runBlocking {
+            super.onCreate(savedInstanceState)
+            setContentView(R.layout.activity_main)
+
+            val textView: TextView = findViewById<TextView>(selftest_output)
+            try {
+                val client = uniffi.xmtpv3.createClient("http://10.0.2.2:5556", false);
```

Comment:
Is this an emulator specific thing? Is there ever a case where it would need to be changed to something like 127.0.0.1?